redaxo/src/addons/structure/plugins/content/boot.php

```diff
@@ -108,5 +108,10 @@
 
 if (!rex::isBackend() || (rex::isBackend() && (rex_be_controller::getCurrentPagePart(1) === 'content' || rex_be_controller::getCurrentPagePart(1) === 'structure'))) {
     rex_minibar::getInstance()->addElement(new rex_minibar_element_structure_article());
+}
+
+$page = rex_be_controller::getCurrentPageObject();
+if (!rex::isBackend() || (rex::isBackend() && $page && !$page->isPopup())) {
```

Comment:
ohh je.. wenn ich diese bedingungen sehe bereue ich dass wir die minibar "aktivierbar" gemacht haben, obwohl sie in popups inaktiv ist.

Reply:
Geht mir genauso. Nur eben, dass diese in Popups generell deaktiviert ist. Ich fände es sinnvoller, es die Elemente selbst entscheiden zu lassen.